app/controllers/ProductController.scala

```diff
@@ -0,0 +1,34 @@
+package controllers
+
+import models.Product
+import play.api.Environment
+import play.api.libs.json.Json
+import play.api.mvc.{AbstractController, ControllerComponents}
+
+import java.time.LocalDateTime
+import javax.inject.Inject
+
+// Controller class for whenever a user enters the URL for the bakery.
+class ProductController @Inject() (
+                                    cc: ControllerComponents,
+                                    environment: Environment
+                                  ) extends AbstractController(cc) {
+  // Create the format for the product in JSON form.
+  implicit val productFormat = Json.format[Product]
+
+  // When the user route to <root>/pass-bakery/status then this will return a status JSON
+  def getAll = Action {
+
+    // Get the current server time when the user enters the link
+    val curDateTime = LocalDateTime.now()
```

Comment:
What is the format of this date? Can you prove it is in the required one?

Reply:
It is ISO 8601 as the documentation says, and as requested from the project. I made sure to include the comment above the code to avoid confusion as to why I used this!